src/dms/core/EdFi.DataManagementService.Core/ApiSchema/ResourceSchema.cs

```
@@ -434,4 +435,26 @@ private readonly Lazy<
     /// The AuthorizationPathways the resource is part of.
     /// </summary>
     public IEnumerable<string> AuthorizationPathways => _authorizationPathways.Value;
+
+    private readonly Lazy<IReadOnlyList<IReadOnlyList<JsonPath>>> _arrayUniquenessConstraints = new(() =>
+    {
+        string? rawText = _resourceSchemaNode["arrayUniquenessConstraints"]?.ToJsonString();
+
+        var stringGroups = JsonSerializer.Deserialize<List<List<string>>>(rawText!);
```

Your logic is a bit off here with the null forgiving operator. If `rawText` is null you will want to throw the same exception on line 450 down below. 

I don't know if the approach of using `ToJsonString()` followed by `Deserialize()` is the most efficient. Did you try using the `.AsArray()` methods like the other elements in this file do?